Dockerfile

```diff
@@ -36,10 +36,8 @@ LABEL maintainer="Microsoft" \
 # libintl and icu-libs - required by azure devops artifact (az extension add --name azure-devops)
 
 # We don't use openssl (3.0) for now. We only install it so that users can use it.
-# Once cryptography is bumped to the latest version, openssl1.1-compat should be removed and openssl1.1-compat-dev
-# should be replaced by openssl-dev.
-RUN apk add --no-cache bash openssh ca-certificates jq curl openssl openssl1.1-compat perl git zip \
- && apk add --no-cache --virtual .build-deps gcc make openssl1.1-compat-dev libffi-dev musl-dev linux-headers \
+RUN apk add --no-cache bash openssh ca-certificates jq curl openssl perl git zip \
+ && apk add --no-cache --virtual .build-deps gcc make openssl-dev libffi-dev musl-dev linux-headers \
```

Comment:
`openssl-dev` can also be removed. I guess this package is added as a dependency of cryptography https://cryptography.io/en/latest/installation/#building-cryptography-on-linux
I leave it here as I don't know if it is used by someone.

Reply:
Perhaps we keep `openssl1.1-compat` for now in case someone is using `openssl1.1` command? I am not sure, though.